src/api/app/views/webui/kiwi/images/show.html.haml

```diff
@@ -10,3 +10,18 @@
     - @image.repositories.each do |repository|
       = render partial: 'repository_entry', locals: {repository: repository}
     = button_to('Edit', edit_kiwi_image_path(@image), method: :get)
+
+%h3 Packages
+- if @image.packages.empty?
+  %p There are no packages.
+- else
+  %table.kiwi_packages_table
+    %tr
+      %th Package
+      %th arch
```

Comment:
I would completely remove these columns and only show for now the package name and maybe a link to details. As you already see from your screenshot, it is very rare that you have additional information / attributes (bootinclude is usually only 2-3 packages, arch is usually the same for all packages etc etc).